bliss/conf/base_config.yaml

```diff
@@ -170,19 +171,38 @@ surveys:
             - run: 94
               camcol: 1
               fields: [12]
+        prior_config: ${prior}
         psf_config:
             pixel_scale: 0.396
             psf_slen: 25
-        n_bands: 5
     decals:
         _target_: bliss.surveys.decals.DarkEnergyCameraLegacySurvey
-        decals_dir: ${paths.decals}
+        dir_path: ${paths.decals}
         sky_coords: # in degrees
             # brick '3366m010' corresponds to SDSS RCF 94-1-12
             - ra: 336.6643042496718
               dec: -0.9316385797930247
         bands: ${encoder.bands}
         reference_band: 1 # DECaLS r-band, DECaLS-indexed (via header)
+        prior_config: ${prior}
```

Comment:
`reference_band` is really a property of the prior, not the survey.

Reply:
Done.